config.py

```diff
@@ -0,0 +1,93 @@
+"""A module for managing environment variables used in GitHub metrics calculation.
+
+This module defines a class for encapsulating environment variables and a function to retrieve these variables.
+
+Classes:
+    EnvVars: Represents the collection of environment variables used in the script.
+
+Functions:
+    get_env_vars: Retrieves and returns an instance of EnvVars populated with environment variables.
+"""
+import os
+from typing import List
+
+
+class EnvVars:
+    # pylint: disable=too-many-instance-attributes
+    """
+    Environment variables
+
+    Attributes:
+        search_query (str): Search query used to filter issues/prs/discussions on GitHub
+        gh_token (str): GitHub personal access token (PAT) for API authentication
+        labels_to_measure (List[str]): List of labels to measure how much time the lable is applied
+        ignore_users (List[str]): List of usernames to ignore when calculating metrics
+        github_server_url (str): URL of GitHub server (Github.com or Github Enterprise)
+        hide_author (str): If set, the author's information is hidden in the output
+        hide_time_to_first_response (str): If set, the time to first response metric is hidden in the output
+        hide_time_to_close (str): If set, the time to close metric is hidden in the output
+        hide_time_to_answer (str): If set, the time to answer discussions is hidden in the output
+        hide_label_metrics (str): If set, the label metrics are hidden in the output
+    """
```

Comment:
I had to disable a linter flag `too-many-instance-attributes` for this to work. 
Should I divide the attributes into multiple objects? for instance `GithubConfig`, `DisplayConfig`?

Reply:
No need to divide attributes, this is fine to keep as you have it.